wandb/sdk/system/assets/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .asset_registry import asset_registry  # noqa: F401
+from .cpu import CPU  # noqa: F401
+from .disk import Disk  # noqa: F401
+from .gpu import GPU  # noqa: F401
+from .gpu_apple import GPUApple  # noqa: F401
+from .ipu import IPU  # noqa: F401
+from .memory import Memory  # noqa: F401
+from .network import Network  # noqa: F401
+from .tpu import TPU  # noqa: F401
```

Comment:
This directory structure violates current standards of putting internal process code under sdk/internal/
is this intentional?

Reply:
fixed!